content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/processors/pinterest.html

```diff
@@ -0,0 +1,39 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-template.render="${@ options}">
+    <a data-pin-do="embedPin" href="${'https://www.pinterest.com/pin/{0}/' @ format=options['pinId']}"></a>
+    <script>
+        var pinterestScript = "//assets.pinterest.com/js/pinit.js";
```

Comment:
There is no good reason to load the `pinterestScript` over http, therefore the script should should start with the https protocol

Reply:
Addressed in 01a691f0e78ded189